templates/designatemdns/config/designate-mdns-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "command": "/usr/bin/designate-mdns --config-dir /etc/designate/designate.conf",
```

Comment:
So should we remove this file? https://github.com/openstack-k8s-operators/designate-operator/blob/main/templates/designate/config/designate-mdns-config.json